### ISSUE TITLE
Make core command failing

### ISSUE THREAD

**Issue by Bradley-Kelman** [jkeger/arctic#42]
Hi all,

I have just tried to install the latest version of arctic from source using the installatiion instructions from the readme but when I come to the make commands I get the following error:
![Screenshot 2023-02-17 103720](https://user-images.githubusercontent.com/125659408/219641305-bafb3bca-5568-409d-8b00-46d7291a03a4.png)

I am not sure if the readme or the code (or even my poor C installation skills!) that need reviewing though.

Thanks!

**Comment by rjmassey** [comment]
Natch. It's a setup difference between linux and mac. I'm guessing you're on linux.
Please edit makefile - comment out line 102 and uncomment line 104. There is only a g different between them.
If you have any idea about how that sort of thing could be made to switch depending on OS, or to use the same on both, I'd love ideas. It's really annoying.
Sorry for the hassle.

**Comment by Bradley-Kelman** [comment]
Yes I'm on linux so that fix will certainly help so thank you. I'm not sure how that would be possible but I can definitely have a think about it. I appreciate your help!

**Comment by kjl0025** [comment]
I had issues with the same command.  I did finally get the package to build, but I had to edit the makefile a decent bit to get it to work on my Mac (which uses brew).  I can try to summarize what I did for my Mac:

I did on my Mac "brew install llvm libomp gsl".  There are some tips at the end of this output which may or may not need to be followed (certain commands to make libraries able to be found, etc).  I added things to my $PATH as it suggested.

I then downloaded the arctic package, and when I got to "make core", I think the makefile assumed I used "port install" instead of "brew".  In my case, I suspect things may have worked fine if I had changed lines 73 and 74 in the makefile from
DIR_OMP ?= $(DIR_MACPORTS)
DIR_GSL ?= $(DIR_MACPORTS)
to 
DIR_OMP ?= $(DIR_HOMEBREW)
DIR_GSL ?= $(DIR_HOMEBREW)

Also, on a Homebrew Mac, one must uncomment line 102, so that it is this (as discussed above on this issue):
LIBS += -L $(DIR_OMP)/lib -lomp
and comment out line 104, so that it is this:
#LIBS += -L $(DIR_OMP)/lib -lgomp

There are some tips in the comments of the makefile that I followed.  There was also an issue with the white-spacing between some paths in the commands in the makefile for me, but this may have been how I defined some path items?  If necessary, a user can change all the path identifiers in the makefile from: 
$(STUFF) 
to:
$(strip $(STUFF))

But I haven't tried a second installation since I don't want to mess up my current one. :) 

Also, a friend of mine could not get this to work on his Linux box.  I am not sure of the details of his attempt, though.

Thanks!
